.github/workflows/build.yml

```diff
@@ -7,37 +7,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: jitterbit/get-changed-files@v1
-        id: files
         with:
-          format: space-delimited
-          token: ${{ secrets.GITHUB_TOKEN }}
+          fetch-depth: 2
       - name: Find changed packages
         run: |
-          function list_include_item {
-            local list="$1"
-            local item="$2"
-            if [[ $list =~ (^|[[:space:]])"$item"($|[[:space:]]) ]] ; then
-              result=0
-            else
-              result=1
-            fi
-            return $result
-          }
-          CHANGED_PACKAGES=()
-          for changed_file in ${{ steps.files.outputs.all }}; do
-            if [[ "$changed_file" == "packages"*".cc" ]] || [[ "$changed_file" == "packages"*".h" ]]; then
-                package=$(echo ${changed_file} | sed 's|\/tizen.*||;s|.*packages\/||')
-                if ! list_include_item $CHANGED_PACKAGES $package; then
-                    CHANGED_PACKAGES+=($package)
-                fi
-            fi
-          done
+          git fetch origin master
+          CHANGED_PACKAGES=$(python3 tools/run_command.py plugins --run-on-changed-packages --base-sha $(git rev-parse HEAD^) 2>&1)
```

Comment:
Question: for `pull_request` events does `HEAD^` return the tip of the target branch?

Reply:
I've confirmed that it does.